test/artefacts/calculate-agricultural-holiday-entitlement/different-number-of-days.txt

```diff
@@ -0,0 +1,12 @@
+What date will your holiday start?
+
+
+
+This is the first day that you will take off work as leave
+
+
+
+
+
+
+
```

Comment:
It doesn't look as though `date_question` options are appearing in the question page artefacts. Is that intentional?